Sources/OpenTelemetryInstrumentationSupport/SpanAttribute+EndUser.swift

```diff
@@ -15,11 +15,14 @@ import TracingInstrumentation
 
 extension SpanAttributeName {
     /// - See: EndUserAttributes
-    public static let endUserID = "enduser.id"
-    /// - See: EndUserAttributes
-    public static let endUserRole = "enduser.role"
-    /// - See: EndUserAttributes
-    public static let endUserScope = "enduser.scope"
+    public enum EndUser {
```

Comment:
`Enduser` since we dont uppercase or separate it in the IDs either? (it is `enduser` not `end_user`)?

Reply:
Mh, was on the fence about this. In the ID we don't have a choice if we want to be Otel-compatible, but grammatically correct would be end-user (at least according to Grammarly 😁).

Reply:
Not a strong opinion, we can go with this 👍